hw3/01-characters.js

```diff
@@ -1,4 +1,26 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const c_list = document.getElementById('charList');
+        fetch(url)
+            .then(response => response.json())
+            .then(characters => {
+                characters.forEach(character => {
+                    const singleChar = document.createElement('div');
```

Comment:
I'm noticing the character boxes aren't all uniform in size, some with extra text in the title underneath the name are a little larger than the cells next to them.  Also adding padding or margin above the photos will add some space around it for the hover color to fully wrap the photo.

![image](https://github.com/rduffy13/rduffy_WebTech_HW/assets/17557205/ab1bfe86-69d6-4c3d-a906-976d27f7bf69)